README.md

```diff
@@ -55,27 +59,30 @@ And here are links of other **HugeGraph** component/repositories:
 3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
 4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)
 
-## Contributing
 
-Welcome to contribute to HugeGraph, please see [`How to Contribute`](CONTRIBUTING.md) for more information.
+## 🤝 Contributing
+
+Welcome to contribute to HugeGraph, please see [`How to Contribute`](CONTRIBUTING.md) & [Guidelines](https://hugegraph.apache.org/docs/contribution-guidelines/) for more information.
 
-## License
+Note: It's recommended to use [GitHub Desktop](https://desktop.github.com/) to greatly simplify the PR and commit process.
 
-HugeGraph is licensed under Apache 2.0 License.
 
-## Thanks
+## 🌹 Thanks
 
 HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we refer to the storage structure of Titan and the schema definition of DataStax. 
 Thanks to TinkerPop, thanks to Titan, thanks to DataStax. Thanks to all other organizations or authors who contributed to the project.
 
 You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.
 
-### Contact Us
+[![contributors graph](https://contrib.rocks/image?repo=apache/hugegraph)](https://github.com/apache/incubator-hugegraph/graphs/contributors)
```

Comment:
wonder to know if we could reduce the size of this contri-graph? (like 50%?)

Reply:
I try it,but it seems to narrow，what do you think？What size it should be, normal size or small size ?
<img width="949" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/38098239/39410418-c3fc-4b91-aa30-1c58d0ad4ae5">

Reply:
> I try it,but it seems too narrow，what do you think？What size it should be, normal size or small size ? <img alt="image" width="949" src="https://user-images.githubusercontent.com/38098239/245114382-39410418-c3fc-4b91-aa30-1c58d0ad4ae5.png">

if we can't choose middle/proper size, seems the normal size is better